### PR TITLE
Update efekta.ts

### PR DESCRIPTION
**Pull request by smartboxchannel**
added support for two new devices EFEKTA PST DUO V1 LR and EFEKTA PST DUO V1

### CODE REVIEW

File: src/devices/efekta.ts [hunk 1]
```diff
@@ -762,6 +769,372 @@ const definitions: DefinitionWithExtend[] = [
...
+                valueOff: ['OFF', 0],
+                cluster: 'genPowerCfg',
+                attribute: {ID: 0x0205, type: 0x10},
+                description: 'Enable сontrol of comparison with previous data',
+            }),
+        ],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpointOne = device.getEndpoint(1);
+            const overrides0 = {min: 7200, max: 43200, change: 1};
+            await reporting.batteryAlarmState(endpointOne, overrides0);
```
Comment (Koenkk) [inline src/devices/efekta.ts:952]: I think this can also be added as a modernExtend?

Comment (smartboxchannel) [reply src/devices/efekta.ts:952]: It would be great, as long as it doesn't work through modernity. As soon as you implement it, I will add efekta.ts

Comment (Koenkk) [reply src/devices/efekta.ts:952]: Could you check if you can add this? Add `batteryAlarmStateReporting` to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/lib/modernExtend.ts#L285

**Comment by smartboxchannel** [comment]
so why was it closed? I initially published a fully working version of the converter.

File: src/devices/efekta.ts (hunk 1 above)
Comment (smartboxchannel) [reply src/devices/efekta.ts:952]: image

Apparently there was something here, but now I don't see anything about batteryAlarmStateReporting.

**Pull request closed without merging**